ocp_utilities/monitoring.py

```diff
@@ -217,3 +202,44 @@ def alerts(self):
         get all the active alerts
         """
         return self._get_response(query=f"{self.api_v1}/alerts")
+
+    def get_alerts_by_state(self, alert_name, state="firing"):
+        """
+        get all the alerts from list of active alerts according the state
+        """
+        alert_list = self.get_all_alerts_by_alert_name(alert_name=alert_name)
+        return [alert for alert in alert_list if alert["state"] == state]
+
+    def wait_for_alert_by_state_sampler(
+        self, alert_name, timeout=TIMEOUT_10MIN, state="firing"
+    ):
+        """
+        Sample output for an alert if found in the state provided in the args.
+
+        Args:
+             alert_name (str): alert name
+             timeout (int): wait time, default is 10 mins
+             state (str): state of the alert to expect, default is firing
+
+        Return:
+             sample (list): list of all alerts that match the alert name and in the state provided in args.
+
+        Raise:
+             TimeoutExpiredError: if alert is not in the state specified before wait_timeout
+        """
+        sampler = TimeoutSampler(
+            wait_timeout=timeout,
+            sleep=self.scrape_interval,
+            func=self.get_alerts_by_state,
+            alert_name=alert_name,
+            state=state,
+        )
+        try:
+            for sample in sampler:
+                if sample:
+                    LOGGER.info(f"Found alert: {alert_name} in {state} state.")
+                    return sample
+
+        except TimeoutExpiredError:
```

Comment:
Please remove try:excpect
No need to do it only for the error log, You will have the info from `TimeoutExpiredError` error